packages/client/hmi-client/src/components/drilldown/tera-ops-preview.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+	<div class="preview">
+		<div class="content-container">
+			<header ref="header">
+				<h5>Preview</h5>
+				<Dropdown
+					v-if="options"
+					class="output-dropdown"
+					v-model="selectedValue"
+					:options="options"
+					@update:model-value="emit('change-output')"
+				></Dropdown>
+			</header>
+			<main :style="{ height: mainHeight }">
+				<slot />
+			</main>
+		</div>
+		<footer>
+			<Button v-if="canSaveAsset" outlined label="Save Model" @click="emit('save-asset')"></Button>
+			<div class="right-aligned-buttons">
+				<Button outlined label="Cancel" @click="emit('cancel')"></Button>
+				<Button label="Apply Changes and Close" @click="emit('apply-changes')"></Button>
+			</div>
+		</footer>
```

Comment:
That `<div class="right-aligned-buttons">` is not necessary, change the footer to be right aligned, and then select the first button to have a `margin-right: auto;`.
Also the goal of that type of footer was to be a component on its own to not repeat it?

```suggestion
		<footer>
			<Button v-if="canSaveAsset" outlined label="Save Model" @click="emit('save-asset')" />
			<Button outlined label="Cancel" @click="emit('cancel')" />
			<Button label="Apply Changes and Close" @click="emit('apply-changes')" />
		</footer>
```

Reply:
hahahahaha proved wrong by my own words, love it 🎉